google/cloud/forseti/notifier/notifiers/slack_webhook.py

```diff
@@ -38,6 +38,12 @@ def _dump_slack_output(self, data, indent=0):
             output: a string formatted violation
         """
         output = ''
+
+        if not isinstance(data, dict):
+            LOGGER.debug('Violation data is not a dictionary type. '
+                         f'Violation data: {data}')
+            return '\t' * (indent + 1) + '`' + str(data) + '`\n'
```

Comment:
Don't throw exception, this prevents other violations from not getting notified.